### ISSUE TITLE
Column Mover - Notify When Changed

### ISSUE THREAD

**Issue by Quantas** [Quantas/shark-ng-table#61]
When clicking the move buttons, there's no notification that anything happened. Ideally, a screenreader user should be notified that a certain column moved.
